pkg/webhook/controlplane/etcd.go

```diff
@@ -170,7 +176,7 @@ func DetermineBackupSchedule(c *corev1.Container, cluster *extensionscontroller.
 			// Randomize the snapshot timing daily but within last hour.
 			// The 15 minutes buffer is set to snapshot upload time before actual maintainance window start.
 			snapshotWindowBegin := maintenanceTimeWindow.Begin().Add(-1, -15, 0)
-			randomMinutes := rand.Intn(59)
+			randomMinutes := int(crc32.ChecksumIEEE([]byte(shootUID)) / 59)
```

Comment:
Shouldn't it to be `% 59`?

```suggestion
			randomMinutes := int(crc32.ChecksumIEEE([]byte(shootUID)) % 59)
```